.eslintrc.json

```diff
@@ -17,7 +17,7 @@
     "dollar-sign"
   ],
   "rules": {
-    "dollar-sign/dollar-sign": [2, "ignoreProperties"],
+    "dollar-sign/dollar-sign": ["error", "ignoreProperties"],
```

Comment:
👍  for "ignoreProperties" because JSCS complains about properties in objects that I can't change.